README.md

```diff
@@ -6,6 +6,7 @@ You fill find useful docs on how to do things specific to the BCGov's implementa
 
 # Index
```

Comment:
Do you want to add some links to documentation here?

I did some additional customization howtos. that I would like to share.. specifically DB tuning changes..

https://access.crunchydata.com/documentation/postgres-operator/5.0.3/tutorial/customize-cluster/
Specifically adding the 

patroni:
  dynamicConfiguration:
    postgresql:
      parameters:
        shared_buffers: 1GB
        work_mem: 2MB

Reply:
Another how to, is patch management.  This needs to be managed by each workspace. using these instructions.
https://access.crunchydata.com/documentation/postgres-operator/5.0.3/tutorial/update-cluster/

Reply:
Added `crunchydb/README.md` with these details